GAE/src/locale/es.properties

```diff
@@ -96,7 +94,7 @@ Choose\ a\ pre-created\ cascade\ from\ the\ Cascade\ resources\ library.\ If\ yo
 Choose\ a\ question\ using\ the\ dropdown\ selectors\ above = Elija una pregunta utilizando el men� desplegable de arriba
 Choose\ an\ existing\ device\ group\ from\ the\ list = Seleccione un grupo de dispositivos existentes de la lista
 Choose\ cascade\ resource = Elija un recurso cascada
-Choose\ survey\ data\ to\ display = Seleccione datos de la encuesta para mostrar
+Choose\ data\ approval\ group = Eligir grupo de aprobaci�n
```

Comment:
"Elija un" or "Elija el" should be fine. "Elija grupo de aprobación" feels _too_ imperative